### PR TITLE
Cognito whiteboard

### PR DESCRIPTION
**Pull request by nnrogers515**
Adds whiteboard and persistent chat/user functions

### CODE REVIEW

File: realscript-app/src/modules/informationBlock.js [hunk 1]
```diff
@@ -11,15 +11,59 @@ class InformationBody extends React.Component {
...
+            </Card.Text>
           </Card.Body>
         </Card>
         <br />
         <Card style={{ backgroundColor: darkMode }}>
           <Card.Body>
-            <Card.Title style={{ color: darkMode_text }}>Announcements/Progress Section</Card.Title>
-            <Card.Text style={{ color: darkMode_text }}>Placeholder</Card.Text>
+            <Card.Title style={{ color: darkMode_text }>About</Card.Title>
+            <Card.Text style={{ color: darkMode_text }>
```
Comment (GabriellaXue) [inline realscript-app/src/modules/informationBlock.js:34]: Missing bracket.

**Review by GabriellaXue — COMMENTED** [review]
Is this whiteboard sharing between clients?

File: realscript-app/src/modules/informationBlock.js (hunk 1 above)
Comment (nnrogers515) [reply realscript-app/src/modules/informationBlock.js]: Thank you for catching that haha

**Comment by nnrogers515** [comment]
The code I am basing this off of is essentially just tutorial code (credited in the code written), we would have to wire up the connections to our server and adjust calls.

**Pull request merged**